Packages/com.unity.inputsystem/InputSystem/Actions/InputActionSetupExtensions.cs

```diff
@@ -1245,9 +1291,9 @@ public BindingSyntax WithInteraction<TInteraction>()
                 if (!valid)
                     throw new InvalidOperationException("Accessor is not valid");
 
-                var interactionName = InputProcessor.s_Processors.FindNameForType(typeof(TInteraction));
+                var interactionName = InputInteraction.s_Interactions.FindNameForType(typeof(TInteraction));
```

Comment:
This looks like an important fix to used interactions list rather than process or list. Looks like this never worked before.